firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionLifecycleNotifier.java

```diff
@@ -108,6 +138,18 @@ void addOnActivityDestroyedListener(@NonNull OnActivityDestroyedListener listene
     }
   }
 
+  void addOnActivityResumedListener(@NonNull OnActivityResumedListener listener) {
+    synchronized (lock) {
+      this.onActivityResumedListeners.add(listener);
+    }
+  }
+
+  void removeOnActivityResumedListener(@NonNull OnActivityResumedListener listener) {
```

Comment:
Could we add comments or java docs to indicate when we should be removing listeners? Otherwise I feel like we could run into the same memory leaks in the future

Reply:
Sounds good. @lfkellogg or @mrichards do either of you have a quick explanation? Otherwise we can discuss over VC